src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -26,14 +26,19 @@ export PAI_DEFAULT_FS_URI={{ hdfsUri }}
 exec 13>$LAUNCHER_LOG_DIR/YarnContainerDebug.log
 BASH_XTRACEFD=13
 
+RUNTIME_YARN_ERR_LOG=$LAUNCHER_LOG_DIR/runtime.yarn.pai.error
+RUNTIME_YARN_LOG=$LAUNCHER_LOG_DIR/runtime.yarn.pai.log
+
+touch RUNTIME_YARN_ERR_LOG
+touch RUNTIME_YARN_LOG
+
 # YARN kill container will send SIGTERM(15) and SIGKILL(9)
 function sigterm_handler()
 {
   printf "[DEBUG] SIGTERM received, job may be killed by YARN\n"
   killed_by_yarn=true
   exit 143
-}
-
+} 1 >> $RUNTIME_YARN_ERR_LOG 2>> $RUNTIME_YARN_LOG
```

Comment:
`1 >> $RUNTIME_YARN_LOG 2>> $RUNTIME_YARN_ERR_LOG`?

Reply:
It's a mistake, thanks for point that